server/README.md

```diff
@@ -22,11 +22,24 @@ To be able to open Keycloak on localhost map port 8080 locally
 
 ## Creating admin account
 
-By default there is no admin user created so you won't be able to login to the admin console. To create an admin account
-you need to use environment variables to pass in an initial username and password. This is done by running:
+To be able to login to the admin console you need to have an admin user.  
+To create an admin account with your login and password you need to use environment variables to pass in an 
+initial username and password. This is done by running:
 
     docker run -e KEYCLOAK_USER=<USERNAME> -e KEYCLOAK_PASSWORD=<PASSWORD> jboss/keycloak
 
+When you run the container without the `KEYCLOAK_USER` and `KEYCLOAK_PASSWORD` variables, the username and password 
+of the admin user are generated automatically. You can find the actual username / password of the admin account with:
+    
+    docker logs <CONTAINER> | grep 'Your first login and password' -A3
```

Comment:
@14DF397FCA Overlooked one issue yesterday yet :(. Please change the ```grep``` search pattern above to match the updated message. E.g. Replace the above command with:

```
docker logs <CONTAINER> | grep 'Username and password of the Keycloak admin account' -A3
```